@Test public void testBadCredentials() throws Exception {
  Configuration conf=new Configuration();
  conf.set(AWS_CREDENTIALS_PROVIDER,BadCredentialsProvider.class.getName());
  try {
    createFailingFS(conf);
  }
 catch (  AccessDeniedException e) {
  }
catch (  AWSServiceIOException e) {
    GenericTestUtils.assertExceptionContains("UnrecognizedClientException",e);
  }
}
